configuration/src/main/java/io/camunda/configuration/Backup.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.configuration;
+
+public class Backup {
+  /** Configuration for backup store S3 */
```

Comment:
nit: consider full service name

```suggestion
  /** Configuration for backup store AWS S3 */
```

Reply:
done